themes/jungfrau/theme.yaml

```
@@ -173,6 +173,8 @@ elements:
             label: ポップアップ画像
           search_results:
             label: 検索結果
+          cd_search_results:
+            label: コンテン検索結果
```

I think it would be better if the label was the same as the movabletype repository.
May I ask you to change it?

https://github.com/movabletype/movabletype/blob/6d70cb7481a616fe92615bdfd49b78cc92ca85ac/plugins/ContentInfoWidget/lib/ContentInfoWidget/L10N/ja.pm#L22

```suggestion
            label: コンテンツの検索結果
```

---

Thanks for pointing that out!

I have corrected string to "コンテンツの検索結果".